### PR TITLE
Pinterest v0.8.0

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
Update pinterest package dependency version

**This PR will result in the following new package version:**

v1.5.0

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**
Update pinterest version in the packages.yml and documentation

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [ ] dbt compile
- [ ] dbt run –full-refresh
- [ ] dbt run
- [ ] dbt test
- [ ] dbt run –vars (if applicable)

Will test once underlying pinterest package gets released, but right now you can switch it out with 

```
  - git: https://github.com/fivetran/dbt_pinterest.git
    revision: feature/pinterest_v5_API_support
    warn-unpinned: false
```

Before marking this PR as "ready for review" the following have been applied:
- [n/a] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [n/a] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

Run with the above mentioned dependency

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [n/a] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [n/a] docs were regenerated (unless this PR does not include any code or yml updates)
Wait until all packages deps updates are merged into the release branch

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:6]: Small suggestion to clearly call out the upstream changes that are included in the upgraded version of Pinterest Ads.

```suggestion
- Pinterest Ads dependency upgraded to v0.8.0. Please see the respective upstream [Pinterest Ads](https://github.com/fivetran/dbt_pinterest/releases/tag/v0.8.0) and [Pinterest Ads Source](https://github.com/fivetran/dbt_pinterest_source/releases/tag/v0.8.0) releases for more details.
```

File: integration_tests/dbt_project.yml
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:93]: Is there a reason these were all renamed? Is it just to be more in line with the actual name of the package? If so, I am on board with that, but remember that the [seed configs for Pinterest](https://github.com/fivetran/dbt_ad_reporting/blob/71a3626968b7a4e261984111457a1b3f905f8964/integration_tests/dbt_project.yml#L286-L375) will also need to be updated to reflect these changes.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-reneeli thanks for getting this started! I have a few comments before this is good to merge into the release branch. Let me know if you have questions. Thanks!

File: integration_tests/dbt_project.yml
Comment (fivetran-reneeli) [reply integration_tests/dbt_project.yml:93]: Yep just to make it standard... thanks for reminding to change it in the configs too!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Thanks for making the updates. LGTM!

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-reneeli I realized the README also needs to be updated to reflect the new version of the Ad Package as well as the Pinterest Ads dependency version upgrades. Would you be able to make those changes and then feel free to merge this into the release branch.

Please hold off on merging the release branch as we are waiting on a number of other updates.

**Pull request merged**
